README.md

```diff
@@ -1,23 +1,41 @@
-# fotogalleri
+# Fotogalleri - FAFA
 
 <p align="center">
     <img
-        src="https://raw.githubusercontent.com/Teknologforeningen/fotogalleri/master/.github/logo_black.svg"
+        src="https://raw.githubusercontent.com/Teknologforeningen/fotogalleri/master/.github/logo_black.svg?sanitize=true"
         alt="Fotogalleri Logo"
-        style="max-width: 100%;"
+        width="200"
     />
 </p>
 
 Photo Gallery for [Teknologföreningen](https://tf.fi)
 
-See live version at: [https://foto.tf.fi](https://foto.tf.fi)
+See live version of the latest release at: [https://foto.tf.fi](https://foto.tf.fi)
+
+---
 
 ## Development
 
 The development tool heavily used is [Make](https://www.gnu.org/software/make/manual/make.html).
 You may find all the commands and variables specified here in the `Makefile`.
 
-### Rules
+### Environmental variables
+
+There are _two_ ways to change environmental variables:
+1. Copy the file `.env.example` to `.env` and change values in the `.env` file.
```

Comment:
Unclear: is .env both a file and a directory? Where are the files/directories located?

Also, should you specify that .env.example can be found in the "fotogalleri"-directory of the project?

Reply:
I prepended `fotogalleri/` to both and changed the wording a bit.

Clearer now?